Dockerfile

```diff
@@ -20,8 +20,9 @@ RUN go build -o /hauser \
     -ldflags "-w -extldflags \"-static\" -X \"main.version=$(cat VERSION)\"" \
     .
 
-FROM scratch
-WORKDIR /
+FROM alpine:3.13.5
+WORKDIR /hauser
+RUN addgroup -S hauser && adduser -S hauser -G hauser && chown -R hauser:hauser /hauser
```

Comment:
Could you do this in the builder image instead and just copy over the directory with owner/perms in tact? Then you could continue to use scratch and have a much smaller final image.

Reply:
I see you already copy the `/etc/passwd` file over from the builder below. I think you might also want to copy over `/etc/group`.

Reply:
So when I first tried this, it didn't work, but it looks like I was missing the `--chown` flag for COPY. I'll update! And good point on adding `/etc/group`!